package.json

```diff
@@ -17,6 +17,7 @@
     "lint:es": "node ../../scripts/eslint",
     "lint:style": "node ../../scripts/stylelint",
     "lint": "yarn run lint:es && yarn run lint:style",
+    "pretest:jest_server": "node ./test/jest_integration/runIdpServer.js &",
```

Comment:
Can this be started during `saml_auth.tests.ts` start up so it only impacts those tests it applies to?

Reply:
yes, this can be done, will only start idp server for the saml tests